pkg/raft/README.md

```diff
@@ -1,6 +1,17 @@
 This package implements a basic consensus protocol for Habitat communities, based off of the HashiCorp Raft library. Habitat can run multiple communities on one node, necesitating multiple instances of a Raft agent to run side by side, which influences most of the major design decisions in this package.
 
+## LibP2P Trasport layer
```

Comment:
besides todo, can we also add long comment summarizing what is currently there and the different parts, how they work together, etc.